lncfg/protocol_integration.go

```diff
@@ -15,10 +20,6 @@ type ProtocolOptions struct {
 	// nodes should always run with them on by default.
 	LegacyProtocol `group:"legacy" namespace:"legacy"`
 
-	// ExperimentalProtocol is a sub-config that houses any experimental
-	// protocol features that also require a build-tag to activate.
-	ExperimentalProtocol
```

Comment:
I think we'd want to leave this in & rather just remove the custom msg stuff out of the `ExperimentalProtocol` struct. We still want to leave the framework for adding experimental features.